src/Csv.php

```diff
@@ -540,6 +540,41 @@ public function auto($file = null, $parse = true, $search_depth = null, $preferr
         return $this->delimiter;
     }
 
+    /**
+     * Get total number of rows in csv without parsing whole data.
```

Comment:
Please add a statement to say that if the CSV is supposed to have a header, it is not counted as a row. Do you think it would be clearer to people if the function would be called `getTotalDataRowCount()`? I suspect the subtlety is not clear to most.

Reply:
Done.

Reply:
Like the new name